intranet/apps/dataimport/management/commands/import_users.py

```diff
@@ -0,0 +1,124 @@
+            username = counselor_data[counselor]["User Name"].strip()
+            if gender == 'M':
+                gender = 'Male'
+            else:
+                gender = 'Female'
+
+            u = get_user_model().objects.get_or_create(username=username,
+                defaults = {
+                    'last_name':last_name,
+                    'first_name':first_name,
+                    'user_type':"counselor",
+                    'gender':(gender=="Male"),   
+                }
+            )
+        teacher_data = data['teachers']
+
+        for teacher in range(len(teacher_data)):
+            gender = teacher_data[teacher]["Gender"].strip()
+            grade = teacher_data[teacher]["Grade"]
+            first_name = teacher_data[teacher]["First Name"].strip()
+            last_name = teacher_data[teacher]["Last Name"].strip()
+            full_name = teacher_data[teacher]["Full Name"].strip()
+            username = teacher_data[teacher]["User Name"].strip()
+            if gender == 'M':
+                gender = 'Male'
+            else:
+                gender = 'Female'
+
+            u = get_user_model().objects.get_or_create(username=username,
+                defaults = {
+                    'last_name':last_name,
+                    'first_name':first_name,
+                    'user_type':"teacher",
+                    'gender':(gender=="Male"),   
+                }
+            )
+
+        student_data = data['students']
+
+        for student in range(len(student_data)):
+            sid = student_data[student]["Student ID"]
+            gender = student_data[student]["Gender"].strip()
+            grade = student_data[student]["Grade"]
+            first_name = student_data[student]["First Name"].strip()
+            last_name = student_data[student]["Last Name"].strip()
+            middle_name = student_data[student]["Middle Name"].strip()
+            full_name = student_data[student]["Full Name"].strip()
+            username = student_data[student]["User Name"].strip()
+            nickname = student_data[student]["Nick Name"].strip()
+            counselor_names = student_data[student]["Counselor"].strip().split(", ")
+            if gender == 'M':
+                gender = 'Male'
+            else:
+                gender = 'Female'
+            
+            counselor = get_user_model().objects.get(last_name=counselor_names[0])
+            if not get_user_model().objects.filter(Q(username=username) | Q(student_id=sid)).exists():
+                get_user_model().objects.create(student_id=sid,
+                    defaults = {
+                        'last_name':last_name,
+                        'first_name':first_name,
+                        'username':username,
+                        'counselor':counselor,
+                        'gender':(gender=="Male"),
+                        'graduation_year':int(username[0:4]),   
```

Comment:
A start index of `0` is assumed
```suggestion
                        'graduation_year': int(username[:4]),   
```